README.md

```diff
@@ -63,7 +63,23 @@ Build the Surefire project using **Maven 3.2.5+** and **JDK 1.8+**.
   ```
   mvn install site site:stage -P reporting,run-its
   ```
-  
+
+* To set up the project in **Eclipse IDE** , please follow these steps:
+
+  * Maven-build (without tests) the `surefire-shared-utils`  module with profile `ide-development` and install it into the local maven repository
+    ```
+    cd surefire-shared-utils
+    mvn clean install -Pide-development -DskipTests
+    ```
+  * Maven-build the `surefire-grouper` module to generate sources into `target/generated-sources/javacc`
+    ```
+    cd surefire-grouper
+    mvn clean verify
```

Comment:
Perhaps rephrase the sentense to:
Build the module `surefire-grouper` using the Maven command `mvn compile -f surefire-grouper/pom.xml` in order to generate and compile sources into `target/generated-sources/javacc`.

Reply:
@Tibor17 
> Perhaps rephrase the sentense ...

Done :)